oci/runtime_v2.go

```diff
@@ -438,6 +439,35 @@ func (r *RuntimeV2) DeleteContainer(c *Container) error {
 
 // UpdateContainerStatus refreshes the status of the container.
 func (r *RuntimeV2) UpdateContainerStatus(c *Container) error {
+	logrus.Debug("Runtimev2.UpdateContainerStatus() start")
+	defer logrus.Debug("RuntimeV2.UpdateContainerStatus() end")
+
```

Comment:
Is a lock needed here to protect this update?

Reply:
No there's no reason for that, I just wanted to get things running/working before I go into the details of what needs a lock. I'll go through that, thanks for the reminder.

Reply:
done